.github/pr-labeler.yml

```diff
@@ -1,5 +1,5 @@
 BC: bc/*
-feature: feature/*
+feature: [feature/*, task/*]
 bug: bugfix/*
```

Comment:
We were also talking about supporting `bug/*` for bugfix type changes

Reply:
Or for introducing bugs? 🙂

Reply:
Or for reintroducing previously fixed bugs :grin:

Reply:
Alright then 🙂. You asked for it, and we listened!